cerbero/utils/__init__.py

```diff
@@ -105,6 +105,13 @@ def to_winepath(path):
         return path
 
 
+def from_winepath(path):
```

Comment:
This method has been discarded on previous merge, just keep a tmp of the original file and remove it, the current code has the example of the usage